app/views/api/v2/smart_proxies/pubkey.json.rabl

```diff
@@ -0,0 +1 @@
+attribute :pubkey
```

Comment:
I wonder if we shouldn't call it `remote_execution_pubkey`, just to be more specific what this key is being used for